command.c

```diff
@@ -931,6 +954,12 @@ void redis_parse_cmd(struct cmd *r) {
                         goto error;
                     }
                     state = SW_KEY_LEN;
+                } else if (redis_bitfield(r)) {
```

Comment:
Since we are handling the commands as argn types this part would not be needed. This part including the`redis_bitfield()` function can be removed as I see it.

Reply:
Indeed. It is superfluous. Removed it and pushed.